docs/components/App.js

```diff
@@ -59,7 +59,7 @@ export default function App({ children }: Props): Node {
           <ColorSchemeProvider colorScheme={colorScheme} id="gestalt-docs">
             <OnLinkNavigationProvider onNavigation={useOnNavigation}>
               <NavigationContextProvider>
-                <AppLayout showResources={isHomePage} colorScheme={colorScheme}>
+                <AppLayout isHomePage={isHomePage} colorScheme={colorScheme}>
```

Comment:
Rather than pass this down as  a prop, can we just read the Context within AppLayout?